### PR TITLE
Removed erroneous Phase 3B online prerequisite from reminder email

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

**Comment by mehalshah** [comment]
LGTM

**Comment by aoby** [comment]
Correction from the PD team: "just to clarify - right now this section appears for Science and Algebra. We want to keep it for Science, but delete it for Algebra."

Modified to fit those requirements.
Referenced constants: https://github.com/code-dot-org/code-dot-org/blob/e8c3d806a4dbfe1349b39d3e612669fad35b6b35/lib/cdo/workshop_constants.rb#L20-L21

File: dashboard/app/views/ops_mailer/_prerequisite_information.html.haml [hunk 1]
```diff
@@ -52,20 +52,20 @@
...
-            Check your progress in Code Studio by clicking the orange button in the top-right corner and selecting Code Studio from the drop down. Then select the PD course you are enrolled in.
-          %p
-            Here’s what a successful online PD looks like. All the progress circles are green.
-          %img{src:"#{image_url}", style:'width: 600px'}
-          %p
-            %em
-              The number of stages/levels may differ from the above image.
-        %li
-          Payment: To be paid, you must complete all the circles in your Code Studio progress bar for the phase and program. Completion is indicated by a green circle (see example below). You will be paid once ALL portions of this PD segment are completed.
+  - if workshop.program_type.to_i == 1 && workshop.phase == 5
```
Comment (mehalshah) [inline dashboard/app/views/ops_mailer/_prerequisite_information.html.haml]: Change 1 and 5 to reference existing constants

**Comment by mehalshah** [comment]
Now LGTM

**Pull request merged**